### DISCUSSION TITLE
Prevent having to hit enter on error?

### DISCUSSION THREAD

**Discussion by 9mm** [neovide/neovide#2695] [Q&A] [answered]
So Rust LSP sometimes goes wild with errors, usually if im typing quickly.

This will cause a scenario where I get stuck in these hellish loops where i have to hit enter, but then that causes another error, and another, and another .. and another

Maybe 10 in a row and i cant get out of it because it keeps requiring hitting enter

I also will mash on command+Q to just hard abort (which is mapped to :q!) but that also will be blocked because its a hotkey that wont run with the warning open

Is there a way i can make it where

Here is me pretending (artificially hard lol to illustrate the problem)

You can also see that im stuck inside an LSP popup that i cant close because escape closes the error and not the popup

https://www.loom.com/share/a56f70204e7940f1850b6b878a9a85c6

**Comment by fredizzimo** [comment, marked as answer]
I don't think there's a way, but the Neovim team is aware https://github.com/neovim/neovim/issues/22478

**Reply by 9mm** [reply]
awesome! thank you
